packages/quick_actions/quick_actions_android/test/quick_actions_android_test.dart

```
@@ -4,169 +4,98 @@
-      const String type = 'type';
-      const String localizedTitle = 'title';
-      const String icon = 'foo';
+  test('clearShortCutItems', () {
+    quickActions.initialize((String type) {});
+    const ShortcutItem item =
+        ShortcutItem(type: 'test', localizedTitle: 'title', icon: 'icon.svg');
+    quickActions.setShortcutItems(<ShortcutItem>[item]);
+    quickActions.clearShortcutItems();
+
+    expect(api.items.isEmpty, true);
+  });
 
-      const ShortcutItem item =
-          ShortcutItem(type: type, localizedTitle: localizedTitle, icon: icon);
+  test('Shortcut item can be constructed', () {
+    const String type = 'type';
+    const String localizedTitle = 'title';
+    const String icon = 'foo';
 
-      expect(item.type, type);
-      expect(item.localizedTitle, localizedTitle);
-      expect(item.icon, icon);
-    });
+    const ShortcutItem item =
+        ShortcutItem(type: type, localizedTitle: localizedTitle, icon: icon);
+
+    expect(item.type, type);
+    expect(item.localizedTitle, localizedTitle);
+    expect(item.icon, icon);
   });
 }
 
-/// This allows a value of type T or T? to be treated as a value of type T?.
-///
-/// We use this so that APIs that have become non-nullable can still be used
-/// with `!` and `?` on the stable branch.
-T? _ambiguate<T>(T? value) => value;
+class _FakeQuickActionsApi implements AndroidQuickActionsApi {
+  List<ShortcutItem> items = <ShortcutItem>[];
+  String? launchAction;
+
+  @override
+  Future<void> clearShortcutItems() {
+    items = <ShortcutItem>[];
+    return Future<void>.value();
+  }
+
+  @override
+  Future<String?> getLaunchAction() {
+    launchAction = LAUNCH_ACTION_STRING;
+    return Future<String?>.value(launchAction);
+  }
+
+  @override
+  Future<void> setShortcutItems(List<ShortcutItemMessage?> itemsList) async {
+    await clearShortcutItems();
+    for (final ShortcutItemMessage? element in itemsList) {
+      if (element != null) {
```

Wouldn't a null here indicate a bug in the code being tested? I would think we would want this to crash, not silently skip.